packages/js/csv-export/src/test/index.ts

```diff
@@ -89,31 +89,23 @@ describe( 'generateCSVFileName', () => {
 
 describe( 'downloadCSVFile', () => {
 	it( "should download a CSV file name to users' browser", () => {
-		const mockFn = jest.fn();
-		jest.spyOn( global, 'Blob' ).mockImplementation(
-			(
-				content?: BlobPart[] | undefined,
-				options?: BlobPropertyBag | undefined
-			) => {
-				return {
```

Comment:
Blob's API got some new properties so this test failed. Seems unnecessary to assert the API of Blob anyway